src/content/web3/index.md

```diff
@@ -0,0 +1,137 @@
+Web3 is the answer to this dilemma. Instead of an internet monopolized by large technology companies, web3 embraces decentralization and is being built, operated, and owned by its users.
+Before we talk about web3, let's explore how we got here.
+
+<Divider />
+
+## The early internet {#early-internet}
+
+Most people think of the internet as a continuous pillar of modern life—it was invented and has just existed since. However, the internet most of us know today is quite different from originally imagined. To understand this better, it's helpful to break the internet's short history into loose periods—web 1.0 and web 2.0.
+
+### What is Web 1.0? {#web1}
+
+In 1989, in CERN, Geneva, Tim Berners-Lee was busy developing the protocols that would become the internet. His idea? To create open, decentralized protocols that allowed information-sharing from anywhere on Earth.
+
+The first inception of the internet, now known as 'Web 1.0', occurred roughly between 1990 to 2004. The internet on Web 1.0 mainly was static websites owned by companies, and there was close to zero interaction between users, leading to it being known as the read-only web.
+
+### What is Web 2.0? {#web2}
+
+The Web 2.0 period began in 2004 with the emergence of social media platforms. Instead of a read-only, the web evolved to be read-write. Instead of companies providing content to users, they also began to provide platforms to share user-generated content and engage in user-to-user interactions.
+As more people came online, a handful of top companies began to control a disproportionate amount of the traffic and value generated on the web.
+
+<Divider />
+
+## Web 3.0 {#web3}
+
+The premise of 'Web 3.0' was coined by Ethereum co-founder Gavin Wood shortly after Ethereum launched in 2014. Gavin put into words a solution for a problem that many early crypto adopters felt: the internet required too much trust. That is, most of the internet that people know and use today relies on trusting a handful of private companies to act in the public's best interests.
+
+### What is web3? {#what-is-web3}
+
+Web3 has become a catch-all term for the vision of a new, better internet. At its core, Web3 uses blockchain, cryptocurrency, and NFTs to give power back to the users in the form of ownership. Chris Dixon said it best: web1 was read-only, web2 is read/write, web3 will be read/write/own.
+
+#### Core ideas of web3 {#core-ideas}
+
+Although it's challenging to provide a rigid definition of is and is not web3, a few core principles guide its creation.
+
+- **Web3 is decentralized:** instead of large swathes of the internet controlled and owned by centralized entities, ownership gets distributed amongst its builders and users.
+- **Web3 is permissionless:** everyone has equal access to participate in web3, and no one gets excluded.
+- **Web3 has native payments:** it uses cryptocurrency for spending and sending money online instead of relying on the infrastructure banks and payment processors.
```

Comment:
```suggestion
- **Web3 has native payments:** it uses cryptocurrency for spending and sending money online instead of relying on the outdated infrastructure of banks and payment processors.
```